addon/initializers/component-styles.js

```diff
@@ -33,13 +33,13 @@ Component.reopen({
   }),
 
   init() {
-    this._super(...arguments);
-
-    const name = this.get('componentCssClassName');
+    let name = this.get('componentCssClassName');
 
     if (this.get('tagName') !== '' && name) {
-      this.classNames.push(name);
+      this.classNames = [name];
```

Comment:
This should be:

```
this.classNames = this.classNames.slice().push(name);
```

Or some variation. Specifically we need the classNames to include existing classes and add the new one.